libs/cardano-ledger-core/src/Cardano/Ledger/Language.hs

```diff
@@ -96,7 +129,8 @@ instance IsLanguage 'PlutusV2 where
 toSLanguage :: forall (l :: Language) m. (IsLanguage l, MonadFail m) => Language -> m (SLanguage l)
 toSLanguage lang
   | fromSLanguage thisLanguage == lang = pure thisLanguage
-  | otherwise = fail $ "Plutus language mismatch. Expected " ++ show thisLanguage ++ ", but got: " ++ show lang
+  | otherwise =
+      fail $ "Plutus language mismatch. Expected " ++ show thisLanguage ++ ", but got: " ++ show lang
```

Comment:
```suggestion
  | otherwise = fail $ "Plutus language mismatch. Expected " ++ show thisLanguage ++ ", but got: " ++ show lang
```

Reply:
This one actually was need, since a line was way too long. Taking this part to another line shaved off 10 characters :slightly_smiling_face: 
Its is still a bit too long, I'll switch it to a multi-line.

Done